File: Day - 1 (Data Types)/try.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {
	
    public static void main(String[] args) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";
		
        Scanner scan = new Scanner(System.in);
        int new_i = Integer.parseInt(scan.nextLine());
        double new_d = Double.parseDouble(scan.nextLine());
        String new_s = scan.nextLine();
        
        int sum_i = i + new_i;
        double sum_d = d + new_d;
        String sum_s = s.concat(new_s);
        
        System.out.println(sum_i);
        System.out.println(sum_d);
        System.out.println(sum_s);
        
        scan.close();
    }
}

File: Day - 1 (Data Types)/try.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int i = 4;
    double d = 4.0;
    char s[] = "HackerRank ";

    int new_i;
    double new_d;
    char new_s[100];  // Assuming a maximum string length of 100 characters

    scanf("%d", &new_i);
    scanf("%lf", &new_d);
    scanf(" %[^\n]s", new_s);  // %[^\n]s to read a line with spaces

    int sum_i = i + new_i;
    double sum_d = d + new_d;
    char sum_s[200];  // Adjust the size based on your needs
    sprintf(sum_s, "%s%s", s, new_s);

    printf("%d\n", sum_i);
    printf("%.1lf\n", sum_d);
    printf("%s\n", sum_s);

    return 0;
}
